lib/ipmi_fru.c

```diff
@@ -4991,24 +4938,22 @@ ipmi_fru_set_field_string_rebuild(struct ipmi_intf * intf, uint8_t fruId,
 	else
 	{
 		printf("Wrong field type.");
-		rc = (-1);
+		rc = -1;
 		goto ipmi_fru_set_field_string_rebuild_out;
 	}
 
 	/*************************
 	3) Seek to field index */
 	for (i = 0;i <= f_index; i++) {
 		fru_field_offset_tmp = fru_field_offset;
-		if (fru_area) {
-			free(fru_area);
-			fru_area = NULL;
-		}
-		fru_area = (uint8_t *) get_fru_area_str(fru_data_old, &fru_field_offset);
+		free_n(&fru_area);
+		fru_area = (uint8_t *)get_fru_area_str(fru_data_old,
+						       &fru_field_offset);
```

Comment:
Here you have a bad mixture of tabs and spaces. Please indent with tabs to the same level as in previous line, then align with spaces. Currently, there are 2 tabs above and 6 tabs plus 7 spaces here.